frontend/apps/filemanager/filemanager.lua

```diff
@@ -41,6 +41,7 @@ local filemanagerutil = require("apps/filemanager/filemanagerutil")
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
 local util = require("ffi/util")
+local FrontUtil = require("util")
```

Comment:
First time we'd see that name "FrontUtil" :) I'd prefer renaming the other for consistency, BaseUtil is used and ok (I prefer ffiutil) - but that would make many unrelated changes (but well, you did it for util.template > T :)

Reply:
Yeah, it felt awkward to me, too, will do ;).